Create Python entry for random.random() (#564)
Codecademy/docs: content/python/concepts/random-module/terms/random/random.md

---

> Title: 'random()'

Reviewer: ```suggestion
Title: '.random()'
```

Description: 'Returns a psuedo-random floating-point number between 0 and 1.'
Subjects:
  - 'Computer Science'
  - 'Data Science'
Tags:
  - 'Random'
  - 'Functions'
CatalogContent:
  - 'learn-python-3'
  - 'paths/computer-science'
  - 'paths/data-science'
---

The `random.random()` method generates a pseudo-random floating-point number between 0 and 1. It takes no parameters.

It can be useful to create a pseudo-random float less than a given number just by multiplying that number by `random.random()`.

> 

Reviewer: Just to follow the general outline of the other "Random Module" term entries. Add a  "Syntax" section before the "Example"  section.

---
Title: '.random()'
Description: 'Returns a pseudo-random floating-point number between 0 and 1.'
Subjects:
  - 'Computer Science'
  - 'Data Science'
Tags:
  - 'Random'
  - 'Functions'
CatalogContent:
  - 'learn-python-3'
  - 'paths/computer-science'
  - 'paths/data-science'
---

> The `random.random()` method returns a pseudo-random floating-point number between 0 and 1. It takes no parameters.

Reviewer 2: ```suggestion
The `random.random()` method accepts no parameters and returns a pseudo-random floating-point number between 0.0 and 1.0.
```

## Syntax

> 

Reviewer 2: ```suggestion

After importing the `random` module, the `random.random()` method can be accessed:

```

Reviewer 2: This is totally not necessary but was wondering everyone else thought about at least mentioning this in the context of applying the syntax?

Author: @Dusch4593, I don't know. Isn't it implied by being an entry under "random-module?"

> It can be useful to create a pseudo-random float less than a given number just by multiplying that number by `random.random()`.

Reviewer 2: ```suggestion
```

I suggest moving this content down to the "Codebyte Example" section.

## Syntax

```py
random.random()
```

> ## Codebyte Example

Reviewer 2: ```suggestion
## Codebyte Example

It is possible to generate a pseudo-random float between 0 and a given number by multiplying it by `random.random()`:
```

> ```codebyte/py

Reviewer 2: ```suggestion
```codebyte/python
```
